site/docs/install-windows.md

```diff
@@ -68,7 +110,7 @@ We recommend installing:
 
     You will need this to build Java code on Windows.
 
-    Also supported: Java 8, 9
+    Also supported: Java 8 and 9
 
 *   [Python 2.7 for Windows x86-64](https://www.python.org/downloads/windows/)
```

Comment:
Can we change the default to Python 3?

Reply:
I'm wary of recommending py3 unless we know it works. Py2 is deprecated but it works.

Which version do we use on CI? If it's py3, that'd give me confidence that we can recommend it.

Reply:
We installed both versions, but use python3 as default on Windows machines. I'm also using Python3 for everyday develop, so it should be fine. But we can also do it after the python 3 migration we're going to do.

Reply:
SGTM, let's defer updating this section.